### PR TITLE
Added helper function for easily creating logged in sessions

### PR DESCRIPTION
**Pull request by jcscottiii**

### CODE REVIEW

File: acceptance/user_login_test.go [hunk 1]
```diff
@@ -44,6 +44,7 @@ var _ = Describe("UserLogin", func() {
 
 		By("allowing the user to click the login button and redirected to fill out the login form and submit it", func() {
 			delayForRendering()
+			delayForRendering()
```
Comment (msecret) [inline acceptance/user_login_test.go]: Is this mean't to be here twice?

**Comment by msecret** [comment]
Just one comment but this looks good.

**Comment by jcscottiii** [comment]
Will make reduced version of this

**Comment by jcscottiii** [comment]
Refer to minimal #115 version

**Pull request closed without merging**